pallets/gear/src/tests.rs

```diff
@@ -4010,6 +4014,158 @@ fn test_async_messages() {
     })
 }
 
+#[test]
+fn missing_functions_are_not_executed() {
+    // handle is copied from ProgramCodeKind::OutgoingWithValueInHandle
```

Comment:
Why do we need so complicated demo for such check?

Reply:
I supposed it doesn't matter how complex test is. Particularly this looks so due to the intention to test absent `handle_reply` function but the original program from `ProgramCodeKind` doesn't meet the test requirements.